src/api/spec/services/workflows/yaml_downloader_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
-RSpec.describe Workflows::YAMLDownloader, type: :service do
-  let(:yaml_downloader) { described_class.new(payload) }
+RSpec.describe Workflows::YAMLDownloader, type: :service, vcr: true do
```

Comment:
VCRs are needed for what exactly? Nothing about this should depend on the backend.

Reply:
It's needed because there is octokit used.

Reply:
Octokit has nothing to do with the Perl backend, it's interacting with GitHub. I'm pretty sure VCRs can be removed for specs of `Workflows::YAMLDownloader` and the specs would still pass. I'm testing this out locally and somehow I'm getting this error for the GitHub spec:

```     
Octokit::Unauthorized:
       GET https://api.github.com/repos/rubhanazeem/hello_world/contents//.obs/workflows.yml: 401 - Bad credentials
```

Reply:
Exactly! VCR can be used for any external service.

Reply:
Because incase of GitLab there is no API call to GitLab.

Reply:
> Exactly! VCR can be used for any external service.

Yeah, I was having a brain fart... hahaha

> Why is it needed for GitHub, but not for GItLab?

I checked again the code and it's all good.